website-custom-views/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -436,17 +449,17 @@ Pass a custom instance of Apollo client, useful when your Custom View has some c
 #### `environment`
 
 **object**\
-Allows to set the `applicationContext.environment`. The passed object gets merged with the tests default environment. Pass `null` to completely remove the `environment`, which renders the `ui` as if no `environment` was given.
+Sets the `customViewContext.environment`. The passed object gets merged with the tests default environment. Pass `null` to completely remove the `environment`, which renders the `ui` as if no `environment` was given.
```

Comment:
@emmenko please double check if `applicationContext` is used in Custom Views. I matched the values available in the `CustomViewContext` and they looked fine.